docs/UserGuide.md

```diff
@@ -15,27 +18,29 @@ can get your product management tasks done faster than traditional GUI apps.
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your computer.
 
-1. Download the latest `Productiv.jar` from [Coming Soon].
+1. Download the latest `Productiv.jar`. [Coming Soon]
 
-1. Copy the file to the folder you want to use as the _home folder_ for Productiv.
+1. Copy the `.jar` file to your preferred folder.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start *Productiv*. 
+The GUI similar to the below should appear in a few seconds. 
+Note that the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`switch contact`** and pressing Enter will switch to the Contact mode.<br>
-   Some example commands you can try:
+1. Type a command in the command box and press Enter to execute it. 
+Here are some example commands you can try:
 
-   * **`switch deliverable`** : Switches to `deliverable` mode.
+   * **`switch`**`deliverable` : Switches to `deliverable` mode.
```

Comment:
Consider adding whitespace between words (e.g **`switch`** `deliverable`...)

Reply:
I just tried and the whitespace doesn't get reflected on markdown. The current implementation should work alright (there's an illusion of whitespace). Thanks for the suggestion 😸 !